### PR TITLE
Build out wireframe UIs

### PR DESCRIPTION
**Pull request by jessepinho**
## Home
image

## Trade
image

## In this PR
- build nonfunctional wireframes for the home/trade screens
- create `<Button />` component
- small tweaks/fixes

### CODE REVIEW

File: react-native-expo/components/Button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { ActionType } from '@/types/ActionType';
+import { Pressable, Sx, Text } from 'dripsy';
+import { ReactNode } from 'react';
+import { GestureResponderEvent } from 'react-native';
+
+export interface ButtonProps {
```
Comment (jessepinho) [inline react-native-expo/components/Button/index.tsx:6]: Copied these over, with their comments, from `penumbra-web` (with some native-specific tweaks).

I haven't copied over _all_ the props yet, as I'm not yet sure how similar the Native vs. Web buttons will be.

File: react-native-expo/components/HomeScreen/index.tsx
Comment (jessepinho) [inline react-native-expo/components/HomeScreen/index.tsx:1]: image

File: react-native-expo/components/TradeScreen/index.tsx
Comment (jessepinho) [inline react-native-expo/components/TradeScreen/index.tsx:1]: image

File: react-native-expo/utils/dripsyTheme.ts [hunk 2]
```diff
@@ -81,7 +81,7 @@ const dripsyTheme = makeTheme({
   fonts: {
     root: 'Poppins',
     mono: 'IosevkaTerm',
-    heading: 'WorkSans',
+    heading: 'Poppins',
```
Comment (jessepinho) [inline react-native-expo/utils/dripsyTheme.ts:84]: Lil tweak I missed when copying over code from web

**Review by TalDerei — APPROVED** [review]
LGTM! cross-referenced the non-functional wireframes for home / trade screens against figma

**Pull request merged**